TeachingRecordSystem/tests/TeachingRecordSystem.SupportUi.Tests/PageTests/Mqs/DeleteMq/IndexTests.cs

```diff
@@ -161,4 +247,15 @@ public async Task Post_Cancel_DeletesJourneyAndRedirects()
             JourneyNames.DeleteMq,
             state ?? new DeleteMqState(),
             new KeyValuePair<string, object>("qualificationId", qualificationId));
+
+    private MultipartFormDataContent CreateFormFileUpload(string fileType)
+    {
+        var byteArrayContent = new ByteArrayContent(new byte[] { });
+        byteArrayContent.Headers.Add("Content-Type", "application/octet-stream");
+
+        var multipartContent = new MultipartFormDataContent();
+        multipartContent.Add(byteArrayContent, "EvidenceFile", $"evidence.{fileType}");
```

Comment:
to better match with suggested param name change above..
```suggestion
        multipartContent.Add(byteArrayContent, "EvidenceFile", $"evidence{fileExtension}");
```

Reply:
Done